source/features/highest-rated-comment.css

```diff
@@ -14,12 +14,16 @@ a.rgh-highest-rated-comment .avatar {
 	left: -55px;
 }
 
-.rgh-highest-rated-comment {
-	--color-auto-gray-1: var(--color-auto-green-1);
-	--color-auto-blue-4: var(--color-auto-green-4);
-	--color-auto-blue-2: var(--color-auto-green-2);
+.rgh-highest-rated-comment:before,
+.rgh-highest-rated-comment:after {
+	--color-bg-info-inverse: var(--color-text-success, #3fb950);
+	--color-bg-tertiary: var(--color-text-success, #3fb950);
```

Comment:
GitHub has hundreds of color variables, we should never need to use hardcoded variables.

Reply:
They are fallbacks

Reply:
I've decided to remove them, let's hope Github doesn't redesign the site anytime soon

Reply:
Resolved in b959d25.